package.json

```diff
@@ -63,9 +63,10 @@
     "prepare": "yarn clean && yarn grammar && yarn fix && yarn check && yarn build",
     "pre-commit": "npm-run-all --parallel ts:changes lint:changes",
     "grammar": "nearleyc src/parser/grammar.ne -o src/parser/grammar.ts",
-    "build:tsup": "tsup src/index.ts --format cjs,esm --sourcemap --dts",
+    "build:cjs": "tsc -p tsconfig.cjs.json && echo \"{\"type\": \"commonjs\"}\" > dist/cjs/package.json",
```

Comment:
Hmm... I think this doesn't work quite as intended. I generates a file containing:

```
{type: commonjs}
```

which isn't valid JSON. I tweaked it a bit: https://github.com/sql-formatter-org/sql-formatter/commit/14e5daff45d1c257d1d6d68da90aa95a1a0c37c1